hardhat.config.ts

```diff
@@ -64,9 +61,6 @@ module.exports = {
       default: 9,
       1: 9,
     },
-    player: {
-      default: 0,
-    },
     oracle: {
```

Comment:
do we even need that? I'll inspect in my next PR.

Reply:
we do
`const { deployer } = await getNamedAccounts();`
we don't need `oracle` though, so I removed it.